core/src/main/scala/akka/persistence/r2dbc/journal/JournalDao.scala

```diff
@@ -205,7 +205,8 @@ private[r2dbc] class JournalDao(journalSettings: R2dbcSettings, connectionFactor
       val result = r2dbcExecutor.updateInBatchReturning(s"batch insert [$persistenceId], [$totalEvents] events")(
         connection =>
           events.foldLeft(connection.createStatement(insertSql)) { (stmt, write) =>
-            bind(stmt, write).add()
+            stmt.add()
+            bind(stmt, write)
```

Comment:
this was because of the change in https://github.com/pgjdbc/r2dbc-postgresql/issues/463